src/octopeer-github/main/Options/DoNotWatchOptions.ts

```diff
@@ -47,17 +47,18 @@ interface CombinationsThatShouldNotBeWatchedTuple extends Array<string | Array<E
 const DoNotWatchOptions = new (class DoNotWatchOptions {
 
     private elementsThatShouldNotBeWatched: ElementThatShouldNotBeWatchedTuple[] = [
-        ["doNotWatchCommentElements", [
+        ["dataComments", [
             ElementID.CONFIRM_INLINE_COMMENT,
             ElementID.CREATE_PR_COMMENT,
             ElementID.EDIT_COMMENT,
         ]],
     ];
 
     private eventsThatShouldNotBeWatched: EventThatShouldNotBeWatchedTuple[] = [
-        ["doNotWatchOnScreenEvents", [EventID.SCROLL_INTO_VIEW, EventID.SCROLL_OUT_OF_VIEW]],
-        ["doNotWatchHoverEvents", [EventID.MOUSE_ENTER, EventID.MOUSE_LEAVE]],
-        ["doNotWatchKeyboardShortcutEvents", [EventID.KEYSTROKE]],
+        ["mouseScrolling", [EventID.SCROLL_INTO_VIEW, EventID.SCROLL_OUT_OF_VIEW]],
```

Comment:
I'm not sure if you need 301 and 302 here or just 300, since Mathias changed the scrolling functionality. Maybe just to be sure, you can put the 300 here as well (after pulling dev into your branch).